pkgs/servers/computing/slurm/default.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, fetchurl, pkgconfig, curl, python, munge, perl, pam, openssl,
-  ncurses, mysql, gtk }:
+  ncurses, mysql, gtk, lua, hwloc, numactl, extraDeps ? [] }:
```

Comment:
What is the expected use of the `extraDeps` parameter ?

I think it would make sense to have explicit (and null-able) parameters for any additional build input